Refactor Inventory Harvester inventory fields

GabrielEValenzuela opened the pull request:
|Related issue|
|---|
|Closes #29892 |

# Description

This PR aims to align the Inventory Harvester with the latest changes introduced in the inventory templates `(wazuh-states-inventory-hardware, interfaces, networks, and protocols)`, following PR #29874. Specifically, it removes deprecated fields like observer, updates internal data structures, and adjusts serialization and test logic accordingly to match the revised schema.

### Testing

```console
================================================================ 70 passed in 1139.51s (0:18:59) ================================================================
```

> [!NOTE]
> For testing make sure that index mapping matches wit #29874 


![hwNew](https://github.com/user-attachments/assets/6add7581-e520-4332-9122-df13a449f168)

![interfaceNew](https://github.com/user-attachments/assets/9b4f957c-8357-441c-9dbb-5d38ee8397f4)

![networkNew](https://github.com/user-attachments/assets/876a1e4a-be62-46f2-b0be-a76accf0ebc6)

![protocolNew](https://github.com/user-attachments/assets/69894902-02ee-4567-ac9a-b24faa055d1d)

pereyra-m reviewed (changes requested):
Gj!
Just a comment about a structure that already exists.


But I've run the QA tests locally, rebasing the branch against the other development and there is one test failing.
Could you check?

```
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:9200 "GET /wazuh-states-inventory-interfaces-cluster01/_search HTTP/1.1" 200 402
DEBUG    test_indexation:test_indexation.py:303 Fetched documents: [{'_index': 'wazuh-states-inventory-interfaces-cluster01', '_id': '001_d4aa8b01955e438235d586585492f3f5edceec1b', '_score': 1.0, '_source': {'agent': {'id': '001', 'name': 'c0ce505227a4', 'version': 'v4.11.0'}, 'host': {'mac': '14:15:6e:37:f2:7e', 'network': {'egress': {'bytes': 52525, 'drops': 0, 'errors': 0, 'packets': 432}, 'ingress': {'bytes': 449599, 'drops': 0, 'errors': 0, 'packets': 661}}}, 'interface': {'mtu': 1500, 'name': 'eth0', 'state': 'up', 'type': 'ethernet'}, 'wazuh': {'cluster': {'name': 'cluster01'}, 'schema': {'version': '1.0'}}}}]
DEBUG    test_indexation:test_indexation.py:304 Expected documents: []
ERROR    test_indexation:test_indexation.py:318 Test failed for 'wazuh_modules/inventory_harvester/qa/test_data/wazuh_db/000_delete_agent': Mismatch in 'wazuh-states-inventory-interfaces-cluster01': expected 0, got 1
assert 1 == 0
 +  where 0 = len([])
ERROR    test_indexation:test_indexation.py:319 Log retained: log_000_delete_agent.out
```

GabrielEValenzuela commented on src/wazuh_modules/inventory_harvester/src/wcsModel/inventoryNetworkHarvester.hpp:
```diff
@@ -19,12 +19,21 @@
+    {
+        std::string_view name;
+
+        REFLECTABLE(MAKE_FIELD("name", &Interface::name));
```
I had that idea, but I don't know if in the future this could change or not, and represent the same `Interface` that `protocol`

pereyra-m replied:
I would have re-used it, but it's just a suggestion.
You can leave it like it is and check if the final reviewers requests changes.

GabrielEValenzuela commented:
@pereyra-m I ran test again

```console
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/001_insert_modify_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/002_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/004_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/005_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/006_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/001_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/002_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/003_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/004_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/005_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/006_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/007_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/000_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/001_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/002_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/003_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/000_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/001_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/002_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/003_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/005_integrity_check_global]
========================================================= 55 passed, 15 deselected in 909.57s (0:15:09) =========================================================

└─$ git log                                                 
commit f530c09b4784f62268a5b750bd75678c0be98c45 (HEAD -> bug/29892_update-inventory-fields)
Author: f-galland <federico.galland@wazuh.com>
Date:   Fri May 23 12:12:10 2025 -0300

    Fine tune templates

commit f0deb8fb0a519f592c3e74a3ebea2cd85b51559a (origin/bug/29892_update-inventory-fields)
Author: Gabriel Valenzuela <gabriel.valenzuela@wazuh.com>
Date:   Tue May 27 14:31:45 2025 -0300

    refactor(inventory-harvester): Remove 'observer' field and update data structures
    
    - Removed the 'observer' field from result and pre-existing JSON files.
    - Updated C++ structures to reflect the removal of 'observer'.
    - Adjusted serialization logic in tests to match the new data layout.
    - Replaced 'observer' with 'interface' in network-related data.
    - Moved serial number directly into the host structure.
```

pereyra-m commented:
@GabrielEValenzuela 

You've run the tests but the one it failed in my environment.

GabrielEValenzuela commented:
# Proof of testing

After the last commit, test are passing

```console
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/002_modify_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/004_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/005_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/file/006_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/002_modify_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/004_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/005_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/fim/registry/006_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/hotfixes/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/001_insert_modify_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/002_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/004_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/005_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/netifaces/006_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/network_address/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/001_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/002_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/os/003_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/003_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/packages/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/000_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/001_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/002_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/003_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/004_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/005_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/006_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/ports/007_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/000_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/001_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/002_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/003_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/processes/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/000_insert_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/001_insert_delete_rsync]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/002_insert_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/003_insert_delete_delta]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/004_integrity_clear]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/inventory/protocols/005_integrity_check_global]
PASSED wazuh_modules/inventory_harvester/qa/test_indexation.py::test_data_indexation[wazuh_modules/inventory_harvester/qa/test_data/wazuh_db/000_delete_agent]
================================================================ 70 passed in 1138.94s (0:18:58) ================================================================
```
